google/cloud/internal/type_traits.h

```diff
@@ -16,6 +16,9 @@
 #define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_INTERNAL_TYPE_TRAITS_H
 
 #include "google/cloud/version.h"
+#include <ostream>
```

Comment:
Probably `#include <iosfwd>` would do in this case.

Reply:
Done.